README.md

```diff
@@ -294,6 +294,7 @@ Environment Variable | Default | Description
 `LINODE_INSTANCE_CACHE_TTL` | `15` | Default timeout of instance cache in seconds
 `LINODE_ROUTES_CACHE_TTL_SECONDS` | `60` | Default timeout of route cache in seconds
 `LINODE_REQUEST_TIMEOUT_SECONDS` | `120` | Default timeout in seconds for http requests to linode API
+`LINODE_EXTERNAL_SUBNET` | `` | Mark private network as external. Example - `172.24.0.0/16`
```

Comment:
nit: this renders a bit weirdly, as two backticks. Do we just want to keep it empty? 
![Screenshot 2024-09-26 at 09 15 57@2x](https://github.com/user-attachments/assets/b0106ebc-409d-408e-bcde-69e117466bdf)